extensions/sql-database-projects/src/tools/packageHelper.ts

```diff
@@ -57,48 +56,19 @@ export class PackageHelper {
 	 */
 	public async addPackageToAFProjectContainingFile(filePath: string, packageName: string, packageVersion?: string): Promise<void> {
 		try {
-			const project = await this.getAFProjectContainingFile(filePath);
+			const project = await aFUtils.getAFProjectContainingFile(filePath);
```

Comment:
`aF` looks a bit strange, in general I'd suggest just spelling out the whole thing (`azureFunctionsUtils`). That isn't too long and removes any ambiguity when reading it